docs/features/message-pumps/service-bus.md

```diff
@@ -147,6 +151,10 @@ public void ConfigureServices(IServiceCollection services)
             // The unique identifier for this background job to distinguish 
             // this job instance in a multi-instance deployment (default: guid).
             options.JobId = Guid.NewGuid().ToString();
+
+            // The timeout when the message pump tries to restart and re-authenticate during key rotation of the connection string.
+            // (default: 5 seconds)
+            options.KeyRotationTimeout = TimeSpan.FromSeconds(10);
```

Comment:
Woops, apperently we've forgot to remove this one from the setings and the docs bc it is now not used anymore.
Hmm, what whould be the best approach here?

Reply:
We could maybe still use this timeout in another way and pass along a cancellation token but not sure if that's added value with the current setup.

Reply:
No worries, I just copied the preview folder but will remove it. What's the right docs for this or was it simply removed?

Reply:
It should have been removed, but it wasn't, so I guess it's also shipped, but doesn't do anything. 😓 .
Hmm, we could remove it from the docs and make it obsolete or something?

Reply:
Fine by me!

Reply:
We can still look at supporting it in a future release as well of course. What's the reason that it's not used?

Reply:
Bc we first used a different approach in restarting the message pump, which the cancellation token played a part in, but with this new appraoch, we actually restart the internal 'kitchen' of the message pump, and then the cancellation token is not needed.

Reply:
So, for this PR, I'll say it's okay?

Reply:
If it's not used I'd remove it as I already did. If the docs are ok for you, feel free to merge!